javascript/javascript1/week1/age_ify.js

```diff
@@ -0,0 +1,21 @@
+// a Future age calculator
+const yearOfBirth=1995;
+const yearOfFuture=2027;
+const age=yearOfFuture-yearOfBirth;
+console.log("You will be "+age+" years old in "+yearOfFuture);  
```

Comment:
Use template literals (modern JavaScript syntax for cleaner code) like  you did in line 16 and 19